### PR TITLE
[tests-only] added function to custom delay

### PR DESCRIPTION
**Pull request by swoichha**
### Description
This PR adds a function `customDelay` to customize the delay and remove the use of nightwatch `client.pause`

### Related Issue
- Fixes https://github.com/owncloud/owncloud-test-middleware/issues/19

### CODE REVIEW

File: src/helpers/webdavHelper.js [hunk 1]
```diff
@@ -42,14 +41,23 @@ exports.delete = async function (userId, file) {
...
 
+  /**
+   * custom sleep
+   */
+  const customDelay = function (time) {
+    return new Promise((resolve) => {
+      setTimeout(resolve, time);
+    });
+  };
+
```
Comment (dpakach) [inline src/helpers/webdavHelper.js:52]: Put this on a different helper file, not webdavHelper. If there is not a appropriate file you can create new one

File: src/stepDefinitions/sharingContext.js [hunk 2]
```diff
@@ -566,7 +567,7 @@ const checkReceivedSharesExpirationDate = function (user, target, days) {
 };
 
 Given(
-  "user {string} has shared file/folder {string} with user {string}",
```
Comment (dpakach) [inline src/stepDefinitions/sharingContext.js:569]: A fix for this is coming, dont change this

**Review by dpakach — CHANGES_REQUESTED** [review]
looks good, just a small change required

**Review by dpakach — COMMENTED** [review]
LGTM :+1:

**Review by individual-it — APPROVED** [review]

**Pull request merged**